content/docs/reference/codebase/_index.md

```diff
@@ -10,4 +10,6 @@ description: >
 
 Autonity is a fork of Geth.
 
-For the base Geth golang reference documentation see [Go Ethereum v1.10.16 <i class='fas fa-external-link-alt'></i>](https://pkg.go.dev/github.com/ethereum/go-ethereum@v1.10.16) on `pkg.go.dev`.
\ No newline at end of file
+For the base Geth golang reference documentation see [Go Ethereum v1.10.16 <i class='fas fa-external-link-alt'></i>](https://pkg.go.dev/github.com/ethereum/go-ethereum@v1.10.16) on `pkg.go.dev`.
+
+For the Solidity reference documentation see [Solidity v0.8.3 <i class='fas fa-external-link-alt'></i>](https://docs.soliditylang.org/en/v0.8.3/index.html).
```

Comment:
We are on `v0.8.19` now, but I don't think we need to give a version here, just link to the latest:

```suggestion
For the Solidity reference documentation see [Solidity<i class='fas fa-external-link-alt'></i>](https://docs.soliditylang.org/en/latest/).
```

Reply:
@cmjc